test/gui/shared/scripts/bdd_hooks.py

```diff
@@ -130,7 +131,10 @@ def hook(context):
         if not os.path.exists(directory):
             os.makedirs(directory)
 
-        pb.savev(os.path.join(directory, filename), "png", [], [])
+        try:
+            pb.savev(os.path.join(directory, filename), "png", [], [])
+        except:
+            pass
```

Comment:
@amrita-shrestha 
There seems to be another issue with the screenshot. So I would recommend not to include fixes for screenshot here but instead create a new issue and investigate the issue and make fix in another PR.

Reply:
ok